services/base-service/spa/client/public/elements/pages/faq/app-faq.tpl.js

```diff
@@ -72,20 +72,19 @@ return html`
         Aggie Experts is a joint pilot project between the Office of the Provost and the UC Davis Library.
         Its purpose is to create a central registry of UC Davis faculty, researchers, experts and creators and showcase the scholarship created at the university.
         Aggie Experts can be used as an expertise discovery platform for finding collaborators, mentors and expert opinions.
-        In the course of two years we will be expanding the registry by adding faculty from the College of Engineering and adjusting its functionality based on user feedback.
         If you have any recommendations, you can contact us by <a href="mailto:lib-experts@ou.ad3.ucdavis.edu">email</a> or submit a request <a href="https://github.com/ucd-library/aggie-experts-public-issues/issues/new/choose">here</a>.
       </li>
-      <li>I am faculty at UC Davis. Why am I not in the registry?</li>
+      <li>I am faculty at UC Davis. Why am I not in Aggie Experts?</li>
```

Comment:
@UcDust could you easily setup your client so it reads this from a file (not in the actual client code, but when it's building the class?  That way VE could just submit PRs for it?